### PR TITLE
feat: add option to install all components at init

### LINKED ISSUE #11: Add an `eject` command that simply installs all components from the project's system.
Resolving this issue involves adding an ejection command that allows for developers to dump all of the specified system's components into the project.

### PR DESCRIPTION
**Pull request by ryanhagerty**
**Summary**

This PR fixes/implements the following:

- Adds option to install all default system components at the start

**How to review this PR**
- [ ] Pull this branch down, run `npm build` and `npm link`
- [ ] Create a new Drupal project somewhere. I used `composer create-project drupal/recommended-project drupal-folder`
- [ ] Go into that folder and run `npm link @emulsify/cli` (make sure there are no warnings/errors)
- [ ] Run `emulsify init "Theme Name"`. You'll see a new option listed that allows an individual to install all components instead of just the default (screen shot below)
- [ ] Install all the components. `emulsify system install compound --all`. To verify, you can go to your theme directory and checkout the components. For example, the `card` is not a default component. 

Screen Shot 2022-02-25 at 9 57 20 AM

**Closing issues**
Closes #11

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ emulsify component install card --force
 
 #### Installing all available components
 
-If you would like to simply install all components available withing the system you've selected, use the `--all` flag.
+If you would like to simply install all components available within the system you've selected, use the `--all` flag.
```
Comment (patrickocoffeyo) [inline USAGE.md:202]: Nice catch! :smile:

**Review by patrickocoffeyo — CHANGES_REQUESTED** [review]
Hey @ryanhagerty! I left a couple of comments here, but overall this looks great!

**Comment by patrickocoffeyo** [comment]
One more quick note... do you think it'd make sense to document the new `--all` parameter in the [usage md](https://github.com/emulsify-ds/emulsify-cli/blob/develop/USAGE.md#system)?

**Comment by ryanhagerty** [comment]
@patrickocoffeyo Thanks for the review! Comments addressed.

**Review by patrickocoffeyo — APPROVED** [review]
@ryanhagerty LGTM, nice!! :raised_hands:

**Pull request merged**